notebooks/time-to-merge-prediction/thoth-station/thoth_station_feature_engg.ipynb

```diff
@@ -12,7 +12,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        elif ttm &lt; 0.0333333:
</pre><p>The time intervals defining the class boundaries look a bit different than the "quantiles" values. Jw, was there a particular reason for choosing these values?</p><p>Would the classes be more "evenly distributed" if we used values closer to the quantile values (like we did in case of origin)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/500/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='97'/>

Reply:
I chose these values to keep these close to the quantile values but also make sure that these are human readable. When the model is being inferred and it gives results on a PR, i thought these would make them more human readable/justifiable.

Reply:
<p>sure thing, I can adjust those class boundaries!</p>

Reply:
Modified them as per your suggestion, the distribution looks much better! And the models also perform slightly better 🎉 Curious, where are you getting these numbers from? the quantile values that we get from the time to merge column look slightly different.

Reply:
<p>Nice! I basically just started with the quantiles, plotted the histogram, and then manually tweaked the boundaries to be more meaningful in human terms and at the same time ensuring they don't skew the hist too much :)</p>